### PR TITLE
Add setup.py and version information

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

**Comment by SylvainCorlay** [comment]
Btw, sphinxify is also a neat name.

**Comment by SylvainCorlay** [comment]
cc- @ccordoba12
ps: I added a question for you in the gitter chat.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+name = 'docrepr'
+here = os.path.abspath(os.path.dirname(__file__))
+
+version_ns = {}
+with open(os.path.join(here, name, '_version.py')) as f:
+    exec(f.read(), {}, version_ns)
+
+setup(
+    name=name,
```
Comment (ccordoba12) [inline setup.py]: `author` is missing. It should be `The Spyder Development Team`.

Comment (SylvainCorlay) [inline setup.py:12]: added

**Comment by ccordoba12** [comment]
Ok, thanks Sylvain!

**Pull request merged**